client/src/components/ProductItems/ProductItems.module.scss

```
@@ -0,0 +1,34 @@
+.ProductItems {
+	* {
+		color: black;
+		margin-left: 2%;
+		margin-right: 2%;
+	}
+	.heartIcon {
+		width: 25px;
+		padding: 7px 0 0 7px;
+	}
+	a {
+		text-decoration: none;
+	}
+
+	div {
+		text-align: center;
+	}
+
+	.itemImages {
+		width: 17vw;
+		height: 13vh;
```

Irgendwie ein bisschen seltsam, dass mit `vw`/`vh` zu machen. meistens lässt man die height absolut unangefasst und regelt dann nur die width. Das erfordert aber, dass der Eltern-Container die gewünschte breite hat.
```suggestion
	.itemImages {
		max-width: 100%;
```

---

```suggestion
	.itemImages {
	        aspect-ratio: 1/1;
		width: 5rem;
		height: auto;
```

---

habe noch ne height: 10rem vergeben, sonst sind die bilder unterschiedlich hoch. so passts aber